api/README.md

```diff
@@ -22,6 +22,12 @@ The backend for this project was built as a .Net Core API using .NET Core. All t
 - Install [Postman](https://www.getpostman.com/)
 - Install [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
 
+### Setup Database
+
+[Create a Cosmos DB instance](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-manage-database-account) in Azure using the SQL API or use the [CosmosDB Emulator](https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator)
+
+Update the `appsettings.Development.json` file with your Cosmos DB Uri and Primary Key in the `ServiceEndpoint` & `AuthKey` settings respectively. This database will initially be empty. If you try out the API with Postman (see below), the collection and sample documents will be created for you automatically. Otherwise it's structure will be created when you create your first list through the frontend.
```

Comment:
I renamed the file, can you add this at beginning of line 29?

```
Copy the contents of the `appsettings.sample.json` into appsettings.Development.json`
```

Reply:
Done - Moved it to top of section as 1st task.